e2e/cypress/tests/integration/enterprise/cloud/billing/billing_history_free_trial_spec.ts

```diff
@@ -0,0 +1,170 @@
+
+// Stage: @prod
+// Group: @cloud_only @cloud_trial
+
+function simulateSubscription() {
+    cy.intercept('GET', '**/api/v4/cloud/subscription/invoices', {
+        statusCode: 200,
+        body: [
+            {
+                id: 'in_1Lz8b0I67GP2qpb43kcuMyFP',
+                number: '8D53267B-0006',
+                create_at: 1667263490000,
+                total: 65,
+                tax: 0,
+                status: 'open',
+                description: '',
+                period_start: 1664582400000,
+                period_end: 1667260800000,
+                subscription_id: 'prod_K0AxuWCDoDD9Qq',
+                line_items: [
+                    {
+                        price_id: 'price_1KLUYiI67GP2qpb48DXFukcJ',
+                        total: 65,
+                        quantity: 0.06451612903225806,
+                        price_per_unit: 1000,
+                        description: 'Cloud Professional',
+                    },
+                ],
+                current_product_name: 'Cloud Professional',
+            },
+            {
+                id: 'in_1LntnKI67GP2qpb4VObu3NgP',
+                number: '8D53267B-0005',
+                create_at: 1664584986000,
+                total: 733,
+                tax: 0,
+                status: 'failed',
+                description: '',
+                period_start: 1661990400000,
+                period_end: 1664582400000,
+                subscription_id: 'prod_K0AxuWCDoDD9Qq',
+                line_items: [
+                    {
+                        price_id: 'price_1KLUZ2I67GP2qpb45uTS89eb',
+                        total: 733,
+                        quantity: 0.7333333333333333,
+                        price_per_unit: 999,
+                        description: 'Cloud Professional',
+                    },
+                ],
+                current_product_name: 'Cloud Professional',
+            },
+            {
+                id: 'in_1LntnKI67GP2qpb4VObu3NgV',
+                number: '8D53267B-0005',
+                create_at: 1664584986000,
+                total: 733,
+                tax: 0,
+                status: 'paid',
+                description: '',
+                period_start: 1661990400000,
+                period_end: 1664582400000,
+                subscription_id: 'prod_K0AxuWCDoDD9Qq',
+                line_items: [
+                    {
+                        price_id: 'price_1KLUZ2I67GP2qpb45uTS89eb',
+                        total: 733,
+                        quantity: 0.7333333333333333,
+                        price_per_unit: 999,
+                        description: 'Cloud Professional',
+                    },
+                ],
+                current_product_name: 'Cloud Professional',
+            },
+        ],
+    });
+    cy.intercept('GET', '**/api/v4/cloud/subscription', {
+        statusCode: 200,
+        body: {
+            id: 'sub_test1',
```

Comment:
For consistency, let this id match the subscription id in the invoices above

Reply:
Got it Allan. Updated the changes